specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/preview/2016-01-01/AzureBridge.json

```diff
@@ -266,6 +270,51 @@
       "items": {
         "$ref": "#/definitions/ProductLink"
       }
+    },
+    "Compatibility": {
+      "description": "Product compatibility",
+      "type": "object",
+      "properties": {
+        "isCompatible": {
+          "description": "Tells if product is compatible with current device",
+          "type": "boolean"
+        },
+        "message": {
+          "description": "Short error message if any compatibility issues are found",
+          "type": "string"
+        },
+        "description": {
+          "description": "Full error message if any compatibility issues are found",
+          "type": "string"
+        },
+        "issues": {
+          "description": "List of all issues found",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CompatibilityIssue"
+          }
+        }
+      }
+    },
+    "CompatibilityIssue": {
+      "description": "Compatibility issue",
+      "type": "string",
+      "enum": [
```

Comment:
No unknown option?

Reply:
Yep. If there are no known issues, product will be marked as compatible :)
And when it is - this array is empty